pkgs/thycotic/dsv-cli/registry.yaml

```diff
@@ -0,0 +1,31 @@
+packages:
+  - type: github_release
+    repo_owner: thycotic
+    repo_name: dsv-cli
+    asset: dsv-{{.OS}}-{{.Arch}}
+    format: raw
+    replacements:
+      amd64: x64
+      arm64: x64
+      386: x86
+      arm64n: arm64
+      windows: win
+    supported_envs:
+      - darwin
+      - amd64
+    checksum:
+      type: github_release
+      asset: 'dsv-{{.OS}}-{{.Arch}}-sha256.txt'
+      file_format: raw
+      algorithm: sha256
+      pattern:
+        checksum: ^(\b[A-Fa-f0-9]{64}\b)
+        file: "(?i)^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
+    link: https://docs.delinea.com/dsv/current/cli-ref/syntax.md
+    description: Delinea DevOps Secrets Vault is a secret vault automation tool for the management of credentials for applications, databases, CI/CD tools, and services.
+    search_words:
+      - secrets
+      - vault
+    files:
+      - name: dsv
+        src: 'dsv-{{.OS}}-{{.Arch}}'
```

Comment:
```suggestion
```

If the format is `raw`, `files[].src` isn't needed

https://aquaproj.github.io/docs/reference/registry-style-guide#if-the-format-is-raw-filessrc-isnt-needed

Reply:
Without that it comes up as `dsv-cli`. I need it to be `dsv`. The files section seemed to rename it correctly. Suggestion?

Reply:
You should specify only `name`.

```yaml
files:
  - name: dsv
```